app/controllers/state_file/questions/nj_tenant_rent_paid_controller.rb

```diff
@@ -6,7 +6,7 @@ class NjTenantRentPaidController < QuestionsController
       before_action -> { @filing_year = Rails.configuration.statefile_current_tax_year }
 
       def self.show?(intake)
-        intake.household_rent_own == 'rent'
+        intake.household_rent_own == 'rent' && StateFile::NjTenantEligibilityHelper.determine_eligibility(intake) == StateFile::NjTenantEligibilityHelper::ADVANCE
```

Comment:
[SAND] this comment is not on the code you edited in this commit, and probably applies in a number of other places, but can you switch from using string comparison to check enum values to using the generated convenience methods, such as `intake.household_rent_own_rent?`? this is a style/safety point that i'm trying to move the CfA team to be consistent on as well, and it's low priority

Reply:
Tracking in separate refactor ticket